Documentation/Input/Pointers.md

```diff
@@ -127,6 +127,19 @@ Touch Started / Updated / Completed | Raised by touch-aware pointers like [`Poke
 > [!NOTE]
 > [`IMixedRealityFocusChangedHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusChangedHandler) and [`IMixedRealityFocusHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusHandler) should be handled in the objects they are raised on. It is possible to receive focus events globally but, unlike other input events, global event handler won't block receiving events based on focus (the event will be received by both global handler and a corresponding object in focus).
 
+#### Pointer input events in action
+
+1. The MRTK input system recognizes an input event has occurred
```

Comment:
It feels like this is a slight duplicate of the stuff already in the input doc above - thoughts on linking/consolidating or otherwise reducing copy pasta?

Reply:
It is duplicated. One points at the other and the other points back. 

I don't know where would be the consolidated position? It's not worth it's own page and there is no all up input events for both pages to be a child of.

Reply:
Can't you instead of copy pasting say something like:

```suggestion
Pointer input events are recognized and handled by the MRTK input system in a similar way than [regular input events](InputEvents.md#input_events_in_action). The difference being that pointer input events are handled only by the GameObject in focus by the pointer that fired the input event - as well as any global input handlers. Regular input events are handled by GameObjects in focus for all active pointers.
```